### PR TITLE
fixes #107

### LINKED ISSUE #107: [BUG] Error when installing cuXfilter from source.
**Describe the bug**
I am following the instructions to build cuXfilter from source. I get the following error when I update conda env using `cuxfilter_dev_cuda10.1.yml`:

ERROR:
```
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound: 
  - cudf[version='>=0.13']
```

**Steps/Code to reproduce bug**

1. Clone the repository and submodules
```
CUXFILTER_HOME=$(pwd)/cuxfilter
git clone https://github.com/rapidsai/cuxfilter.git $CUXFILTER_HOME
cd $CUXFILTER_HOME
```
```
2. conda create --name cuxfilter_dev 
```
```
3.  conda activate cuxfilter_dev 
```
4. install cudf via 
```
conda install -c rapidsai-nightly -c nvidia -c conda-forge \
    -c defaults cudf=0.13 python=3.7 cudatoolkit=10.1
```
5.  Update conda env
```
conda env update --file conda/environments/cuxfilter_dev_cuda10.1.yml
```
That step generates the error above.

**Expected behavior**

`rapidsai-nightly` is missing in the channel list in `cuxfilter_dev_cuda10.1.yml`,  and `name` does not match with the env name- cuxfilter_dev. But even though I perform these updates, I still get the same error.

We should install cuXfilter from source without any issue.

**Environment details (please complete the following information):**
 - Environment location: [conda]
 - Method of cuxfilter install: [from source]
   - If method of install is [Docker], provide `docker pull` & `docker run` commands used
 

**Additional context**

### PR DESCRIPTION
**Pull request by AjayThorve**

### CODE REVIEW

File: conda/environments/cuxfilter_dev_cuda10.0.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # name: cudf_dev10.0
 channels:
-- rapidsai
+- rapidsai-nightly
```
Comment (raydouglass) [inline conda/environments/cuxfilter_dev_cuda10.0.yml]: You should leave `rapidsai` in this file above `rapidsai-nightly`. For example, this is how cudf does it: https://github.com/rapidsai/cudf/blob/branch-0.13/conda/environments/cudf_dev_cuda10.0.yml

File: conda/environments/cuxfilter_dev_cuda10.0.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 - nvidia
 - conda-forge
 - anaconda
 dependencies:
 - cupy>=6*
 - python>=3.6,<3.8
-- cudf>=0.13
+- cudf=0.13
```
Comment (raydouglass) [inline conda/environments/cuxfilter_dev_cuda10.0.yml]: Please also update https://github.com/rapidsai/cuxfilter/blob/branch-0.13/ci/release/update-version.sh#L55 so during release, the `>` isn't added back.

File: conda/environments/cuxfilter_dev_cuda10.0.yml (hunk 1 above)
Comment (AjayThorve) [reply conda/environments/cuxfilter_dev_cuda10.0.yml]: updated it, thanks!

File: conda/environments/cuxfilter_dev_cuda10.0.yml (hunk 2 above)
Comment (AjayThorve) [reply conda/environments/cuxfilter_dev_cuda10.0.yml]: done

**Comment by dillon-cullinan** [comment]
rerun tests

**Pull request merged**